thirdeye/thirdeye-frontend/app/pods/components/filter-bar/component.js

```diff
@@ -225,6 +247,20 @@ export default Ember.Component.extend({
     });
   },
 
+  /**
+   * Counts the number of events of a given event type
+   * @param eventType - string
+   * @return {Number}
+   */
+  countEvents(eventType) {
```

Comment:
It seems like all these events are updated at once whenever 'entities'[] is updated. If so, this way of counting the entities could be arguably more clear but seems really roundabout and inefficient since you're counting through the same 'entities'[] for each eventType multiple times. Wouldn't it be easier to just count all the events at once in a single function that loops through 'entities'[] once and keeps track of the count for each type in the object you return instead of having to define two functions where one of them just calls the other function repeatedly?

Reply:
Ooh that's a good point. I'll make the change. Thanks!